src/mots/cli.py

```diff
@@ -156,6 +157,52 @@ def export(args: argparse.Namespace) -> None:
             f.write(output)
 
 
+def set_default(args: argparse.Namespace):
+    """Set a specified settings variable to the provided value."""
+    key = args.key[0]
+    value = args.value[0]
+    with settings.OVERRIDES_FILE.open("r") as f:
+        overrides = yaml.load(f) or {}
+
+    if key not in settings.DEFAULTS:
+        raise ValueError(f"{key} does not exist in defaults.")
+
+    _type = type(settings.DEFAULTS[key])
+
+    if _type(value) == overrides.get(key):
+        raise ValueError(f"{key} is already set to {value} in overrides file.")
+
+    overrides[key] = _type(value)
+    print(f"{key} is now set to {value} ({_type.__name__}) in overrides file.")
+
+    with settings.OVERRIDES_FILE.open("w") as f:
+        yaml.dump(overrides, f)
```

Comment:
Note that this doesn't preserve comments or the order of the file before the new default is set. Maybe that doesn't matter but it is a potential footgun.

Reply:
This uses the same comment-preserving and round-trip preservation mechanisms as with `mots.yaml`, thanks to `ruamel.yaml` (`yaml` here is `mots.yaml.yaml`).

Reply:
I tested this but in an odd way - you're right it does preserve comments.